mithril.js

```diff
@@ -49,9 +49,12 @@ var m = (function app(window, undefined) {
 	 *
 	 */
 	function m(tag, pairs) {
-		for (var args = [], i = 1; i < arguments.length; i++) {
-			args[i - 1] = arguments[i];
+		// JSX compatibility with 'null' {pairs} argument
+		var i = pairs === null ? 2 : 1;
+		for (var args = []; i < arguments.length; ++i) {
+			args.push(arguments[i]);
 		}
+
 		if (isObject(tag)) return parameterize(tag, args);
```

Comment:
Nit: shouldn't `pairs` be sent as-is as well to `parametrize`, in the form of the first part of `args`? Something like this?

```js
for (var args = [], i = 0; i < arguments.length; i++) {
  args.push(arguments[i]);
}

if (isObject(tag)) return parameterize(tag, args);
if (pairs === null) args.shift();
```